homeassistant/components/deconz/climate.py

```diff
@@ -187,7 +215,9 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
             raise ValueError(f"Unsupported HVAC mode {hvac_mode}")
 
         data = {"mode": self._hvac_mode_to_deconz[hvac_mode]}
-        if len(self._hvac_mode_to_deconz) == 2:  # Only allow turn on and off thermostat
+        if (
+            len(self._hvac_mode_to_deconz) == 2 and not self.auto_to_heat_mapping
+        ):  # Only allow turn on and off thermostat
```

Comment:
```suggestion
        if len(self._hvac_mode_to_deconz) == 2:  # Only allow turn on and off thermostat
```

len(self._hvac_mode_to_deconz) is 3 for SPZB0001